js/ui/overlay/ui.overlay.js

```diff
@@ -432,8 +435,18 @@ const Overlay = Widget.inherit({
     },
 
     _renderWrapperAttributes() {
-        const { wrapperAttr } = this.option();
-        this._$wrapper.attr(wrapperAttr ?? {});
+        const { wrapperAttr } = this.option() || {};
```

Comment:
Is `||` necessary here? When does `this.option()` give us a nullish value?

Reply:
yep, not necessary